agent.go

```diff
@@ -63,9 +64,13 @@ type Payload struct {
 
 // ProbeResult structure for network probing results
 type ProbeResult struct {
```

Comment:
AFAIC, even for the server, it would not be very useful as these values are updated frequently instead of counters that should accumulate events for an infinite span of time.